package.json

```diff
@@ -21,7 +21,6 @@
   },
   "homepage": "https://github.com/orbiting/poc-pdf#readme",
   "dependencies": {
-    "@react-pdf/core": "^0.6.3",
```

Comment:
On purpose? Because you use a locally linked version?

Reply:
Exactly. This Is going to point to a new version after we release the page wrapping feature to npm.
Speaking of that, @tpreusse  what do you think if we merge this feature? It covers the main use case at least. I didn't do much testing with much complicated layouts, but it's a good start

Reply:
Release the beast! IMO go for it, getting it into more hands will make it better faster, React PDF is pre v1 – people should be fine with a bit of turbulence. And I also agree it's a great start.

Reply:
Awesome. The new one is going to be `0.7.0`. A feature like this deserves a minor update!